src/common/helpers/auth/get-jwt-strategy-config.js

```diff
@@ -0,0 +1,73 @@
+import Boom from '@hapi/boom'
+import { getDefraUserRoles } from './get-defra-user-roles.js'
+import { getEntraUserRoles } from './get-entra-user-roles.js'
+import { config } from '../../../config.js'
+
+export function getJwtStrategyConfig(oidcConfigs) {
+  const { entraIdOidcConfig, defraIdOidcConfig } = oidcConfigs
+
+  return {
+    keys: [
+      {
+        uri: entraIdOidcConfig.jwks_uri
+      },
+      {
+        uri: defraIdOidcConfig.jwks_uri
+      }
+    ],
+    verify: {
+      aud: false,
+      iss: false,
+      sub: false,
+      nbf: true,
+      exp: true,
+      maxAgeSec: 3600, // 60 minutes
+      timeSkewSec: 15
+    },
+    validate: async (artifacts, request) => {
+      const tokenPayload = artifacts.decoded.payload
+      const { iss: issuer, aud: audience, id: contactId, email } = tokenPayload
+
+      if (issuer === entraIdOidcConfig.issuer) {
+        // Entra Id is not providing an audience in the token, so we need to supply it
+        // This audience may not need to be a secret, only an env var
+        const adminUiAsAudience = config.get('oidc.entraId.audience')
+        if (audience !== adminUiAsAudience) {
+          throw Boom.badRequest('Invalid audience for Entra ID token')
```

Comment:
opinion: I'd opt for returning `{ isValid: false }` so that a hapi/jwt can respond with 401/403 (I think throwing `badRequest` here will result in a 400)